### PR TITLE
[ADF-5031][PeopleCloudComponent] [PeopleCloudComponent] Provide a way to make selected items read-only (not remove)

### PR DESCRIPTION
**Pull request by sivakumar414ram**
**Please check if the PR fulfills these requirements**

> - [x] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [x] Tests for the changes have been added (for bug fixes / features)
> - [x] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [ ] Bugfix
> - [x] Feature
> - [ ] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)

https://issues.alfresco.com/jira/browse/ADF-5031

**What is the new behaviour?**

ezgif com-video-to-gif (5)

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: docs/process-services-cloud/components/people-cloud.component.md [hunk 1]
```diff
@@ -29,6 +29,7 @@ Allows one or more users to be selected (with auto-suggestion) based on the inpu
 | preSelectUsers | [`IdentityUserModel`](../../../lib/core/models/identity-user.model.ts)`[]` |  | Array of users to be pre-selected. All users in the array are pre-selected in multi selection mode, but only the first user is pre-selected in single selection mode. Mandatory properties are: id, email, username |
 | roles | `string[]` |  | Role names of the users to be listed. |
 | searchUserCtrl | `FormControl` | new FormControl() | FormControl to search the user |
+| removeLastChip | `Boolean` | true | This flag enables not to delete last chip in multiple mode. In case the flag is false, last chip from the list can not be deleted  Otherwise, last chip can be deletable. |
```
Comment (eromano) [inline docs/process-services-cloud/components/people-cloud.component.md]: I am not sure about the name of this property.

Comment (mauriziovitale) [inline docs/process-services-cloud/components/people-cloud.component.md]: I don't like this strategy. As I said we should be able to pass a readonly property(true/false) for the preselected users

Comment (sivakumar414ram) [reply docs/process-services-cloud/components/people-cloud.component.md]: This property no longer needed and removed. we have changed the strategy to add readonly property to the pre-selected items.

Comment (sivakumar414ram) [reply docs/process-services-cloud/components/people-cloud.component.md]: Done

**Pull request merged**